public_html/views/index.php

```
@@ -0,0 +1,203 @@
+            </tr>
+        </thead>
+    HTML;
+}
+
+function render_total_by_year($years_data): array
+{
+    $total_by_year = [];
+    foreach ($years_data as $year => $data) {
+        // sum all data values for this year
+        $y_count =  0;
+        foreach ($data as $lang => $count) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$lang'. (undefined)

(UnusedLocalVariable)